docs/book/v6/tutorials/create-book-module.md

```diff
@@ -754,7 +825,7 @@ To list the books use:
 curl http://0.0.0.0:8080/books
 ```
 
-To retrieve a book use:
+To retrieve a book, `curl` one of the links found in the output of the **list books** command, under `_embedded` . `books` . * . `_links` . `self` . `href`
```

Comment:
```suggestion
To fetch a book, `curl` one of the links found in the output of the **list books** command, under `_embedded` . `books` . * . `_links` . `self` . `href`.
The link should have the following format:
```
I replaced `retrieve` with `fetch`, because the latter is the more common term used when obtaining data via an API.
Also, added the last sentence so that the below curl snippet makes more sense.